### DISCUSSION TITLE
Webinar: getting the most from Artisan's tools on October 6 at 1pm PST (22pm CEST)

### DISCUSSION THREAD

**Discussion by MAKOMO** [artisan-roaster-scope/artisan#954] [Announcement]
Join us on October 6 at 1pm PST (22pm CEST) for the second Royal Coffee webinar on Artisan, where Dave will do a deep dive into the Analyzer and the Comparator tools.
[Registration required!](https://artisan-scope.org/event/webinar/Royal-Coffee-Webinar-2/)

![2022-10-06-Royal-Coffee-Webinar2](https://user-images.githubusercontent.com/13909011/189478251-f0121450-f969-4a93-b2e8-2a087167f4c2.png)

**Comment by MAKOMO** [comment]
Remember to register for tomorrows RoyalCoffee webinar on Artisan!

**Comment by fatrabbit-la** [comment]
I’m registered but have to work now. How will we be able to view it later?Sent from the SuperComputer in my pocketOn Oct 4, 2022, at 11:05 PM, Marko Luther ***@***.***> wrote:﻿
Remember to register for tomorrows RoyalCoffee webinar on Artisan!

**Reply by MAKOMO** [reply]
We hope that RoyalCoffee will again provide a recording that we can publish on the [Artisan YouTube channel](https://www.youtube.com/channel/UCEp-DH_Y7sBR3-X3FWomgog).

**Comment by MAKOMO** [comment]
[The recording of this webinar](https://www.youtube.com/watch?v=8ivsccu9e_Y) is now available on our [YouTube channel](https://www.youtube.com/channel/UCEp-DH_Y7sBR3-X3FWomgog), including section marks and subtitles. Enjoy!
